### DISCUSSION TITLE
Miss port specified

### DISCUSSION THREAD

**Discussion by fajarmuslim** [bentoml/BentoML#1585] [Q&A]
I try to run bento service with docker command, but when, trying to used port other than 5000. I faced the logs showing that the service still run on the 5000 port. How to solve this? Thanks in advance

command : `docker run -p 5001:5001 catboost_apartment`
logs : [2021-04-19 10:20:46 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)

**Comment by parano** [comment]
@fajarmuslim `-p 5001:5001` is a parameter for docker, you also need to pass a parameter to BentoML to let BentoAPIServer run on port 5001. You can do that with:

```bash 
docker run -p 5001:5001 catboost_apartment --port 5001
```

alternatively with env var:
```bash 
docker run -p 5001:5001 -e BENTOML_PORT=5001 catboost_apartment 
```

**Reply by fajarmuslim** [reply]
thanks a lot @parano , the first solution was working properly, but the env var BENTOML_PORT doesn't work for me and it is good to use first solution

**Reply by parano** [reply]
that's unexpected, might be a bug on our end, I will look into it. thanks for the feedback @fajarmuslim

**Reply by fajarmuslim** [reply]
cool!!
